From: Michael Davidsaver <mdavidsaver@gmail.com>
Date: Mon, 4 Apr 2016 22:10:29 -0400
Subject: RELEASE

---
 configure/RELEASE | 8 +++-----
 1 file changed, 3 insertions(+), 5 deletions(-)

diff --git a/configure/RELEASE b/configure/RELEASE
index 0f737f5..7d840d2 100644
--- a/configure/RELEASE
+++ b/configure/RELEASE
@@ -10,18 +10,16 @@
 
 TEMPLATE_TOP=$(EPICS_BASE)/templates/makeBaseApp/top
 
-SUPPORT=/corvette/home/epics/devel
--include $(TOP)/../configure/SUPPORT.$(EPICS_HOST_ARCH)
+SUPPORT=$(EPICS_BASE)
 
-ASYN=$(SUPPORT)/asyn-4-28
+ASYN=$(EPICS_BASE)
 
 # If you don't want to install into $(TOP) then
 # define INSTALL_LOCATION_APP here
 #INSTALL_LOCATION_APP=<fullpathname>
 
 # EPICS_BASE usually appears last so other apps can override stuff:
-EPICS_BASE=/corvette/usr/local/epics/base-3.14.12.5
--include $(TOP)/../configure/EPICS_BASE.$(EPICS_HOST_ARCH)
+EPICS_BASE=/usr/lib/epics
 
 #Capfast users may need the following definitions
 #CAPFAST_TEMPLATES=
